### PR TITLE
No js translations

### LINKED ISSUE #5321: Provide translations for no-javascript scenarios in the media player
**Is your feature request related to a problem? Please describe.**
We need translations for all services when a user has disabled javascript and tries to view a page containing media.

**Describe the solution you'd like**
When javascript is disabled and I view a Simorgh page containing a media player I can see a message, relative to the service, indicating I need to enable javascript to view the media.

**Testing notes**

1. Disable javascript on your browser.
2. Go to a page containing a media player (video/audio)
3. Observe localised error message

**Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc**

- Translations are available in the service configs e.g. `src/app/lib/config/services/burmese.js` in the `media` object with the key name `noJs`.
- [Afrique translation](https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=757630729&range=S83) needs added to [service config](https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/afrique.js).
- Be mindful of services with variants e.g. `zhongwen`
- Discuss whether to follow this convention https://github.com/bbc/simorgh/pull/5329#discussion_r373558921

- [x] This feature is expected to need manual testing.

**Additional context**
Translations can be found here (Row 83): https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=757630729

We must use the Burmese Unicode translation and not Burmese Original.

If there is no translation available for a service then we should fallback to the English version.

### PR DESCRIPTION
**Pull request by jroebu14**
Partially resolves https://github.com/bbc/simorgh/issues/5321

**Overall change:** 
Adds translations for the message displayed in the media player when javascript is disabled.

**Code changes:**
- Adds translations to the service configs in `src/app/lib/config/services/` e.g. `burmese.js`
- The translations are part of the `media` object with the key name `noJs`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: add-no-translations.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
```
Comment (karinathomasbbc) [inline add-no-translations.js]: Can we move this to the scripts directory & ignore in .codeclimate.yml?

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

**Comment by karinathomasbbc** [comment]
Also, a general question - if there is no text for the no-js message, will it fall back to a default message? Or should we include the default message in the config for each service?

File: add-no-translations.js (hunk 1 above)
Comment (jroebu14) [reply add-no-translations.js]: I’m going to delete this file when I’m finished working on it. I open draft PRs as early as possible so this is not production ready yet! I’ll pop it into the code review column when I need keen eyes on it but still feel free to question the general direction. The code will most likely change a bit though

**Comment by jroebu14** [comment]
https://github.com/bbc/simorgh/pull/5329#issuecomment-580311742 Yes it will fall back to English. Need to work on that mechanism tgouhh

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/lib/config/services/afrique.js [hunk 2]
```diff
@@ -139,6 +139,7 @@ export const service = {
         },
       },
       media: {
+        noJs: '',
```
Comment (ryanmccombe) [inline src/app/lib/config/services/afrique.js:142]: I think we normally have english in these configs until we get translations?  This is fine though if we have the fallback elsewhere 👍

**Review by ryanmccombe — APPROVED** [review]

File: src/app/lib/config/services/afrique.js (hunk 2 above)
Comment (jroebu14) [reply src/app/lib/config/services/afrique.js:142]: Ah ok. We can follow that convention when we come to plumbing the no js messages into the media player.

**Pull request merged**